### PR TITLE
Added support for having coding standards in the root package

### PR DESCRIPTION
**Pull request by frenck**
## Proposed Changes

This change adds support for having PHP_CodeSniffer standards in the root package.

## Related Issues

Implements: #20 Check the root package for sniffs to install

### CODE REVIEW

**Comment by frenck** [comment]
Travis tests fails, enabled by @Potherca, but not configured it yet.
Therefore the failed Travis builds can be ignored for now.

File: src/Plugin.php [hunk 1]
```diff
@@ -261,23 +261,26 @@ private function cleanInstalledPaths()
     private function updateInstalledPaths()
     {
         $changes = false;
-        $codingStandardPackages = $this->getPHPCodingStandardPackages();
 
+        $searchPaths = [getcwd()];
```
Comment (Potherca) [inline src/Plugin.php]: For completeness sake: although usually discouraged using `getcwd` (and consorts) is the "Composer Way" of doing things.

**Comment by frenck** [comment]
Just thought of an edge case that might cause troubles with this solution:
- This plugin is installed globally
- Current working directory is a repository containing a coding standard
- Running a composer global install/update command

This might cause the installation of the local coding standard into the global scope.

Will check on this tomorrow.

**Comment by frenck** [comment]
Check my above concerns and concluded this won't be an issue since the Composer GlobalCommand will change the working directory _before_ actually running Composer.

See: https://github.com/composer/composer/blob/9e7d1a4f9ed34466a05ac1356e97a98686f7da67/src/Composer/Command/GlobalCommand.php#L77

**Pull request merged**